webapp/src/components/List/List.spec.js

```diff
@@ -0,0 +1,37 @@
+import {mount} from '@vue/test-utils'
+import List from '@/components/List/List.vue'
+
+describe (
+    "List.vue", ()=> {
+
+        test("Test if List is empty",()=> {
+            const wrapper = mount(List,
+                {
+                    propsData:
+                {
+                    listItems:[]
+                }
+            
+            })
+            expect(wrapper.html()).toContain("list is empty :(")
+        })
+        test("Test if list is not empty",()=> {
+            const wrapper = mount(List,
+                {
+                    propsData:
+                {
+                    listItems:[{
+                        
+                            id: 0,
+                            title: "VueJSTest",
+                            votes: 3,
+                            },
+                    ]
+                }
+            
+            })
+            expect(wrapper.html()).not.toContain("list is empty :(")
```

Comment:
I'd rather check whether `listItems` is correctly loaded, e.g. check whether `title` for example is contained.

Reply:
Yes, testing for nothingness is really error-prone and can lead to silently passing, false-negative tests.